Dockerfile

```
@@ -21,18 +38,12 @@ RUN tmp="$(mktemp -d)" set -x \
 
 COPY package.json package-lock.json /usr/src/app/
 RUN npm ci --unsafe-perm --production && npm cache clean --force 2>/dev/null
-COPY . /usr/src/app
 
-COPY openvpn /etc/openvpn
+COPY src /usr/src/app/src
+COPY --from=builder /usr/src/app/build /usr/src/app/build
+COPY bin /usr/src/app/bin
```

I think you can condense these with
```suggestion
COPY src bin config openvpn /usr/src/app/
```

---

Unfortunately not, that copies the contents of the directories.

---

Ah
```suggestion
COPY src/ bin/ config/ openvpn/ /usr/src/app/
```

---

Same result, afaik it isn't possible to copy multiple directories with a single `COPY`.